index.html

```diff
@@ -578,12 +711,26 @@ <h2>Self-description</h2>
             <p>
                 The self-description is an exploration mechanism in which a <a>Thing</a>
                 hosts its own <a>TD</a> and exposes it at a URL or
-                through others means. 
+                through other means. 
                 If exposed at a URL (e.g. over HTTP or CoAP), the URL may be advertised
                 via one of the [[[#introduction-mech]]].
-                The hosted <a>TD</a> may also be registered inside a <a>Thing Description Directory</a>
+                The same <a>TD</a> may also be registered (at different URL) inside a <a>Thing Description Directory</a>
                 as prescribed in [[[#exploration-directory]]]. 
             </p>
+	    <p>
+	    In some cases a single URL may need to return multiple
+	    TDs.  For large numbers of TDs that may need to change
+	    dynamically over time, 
+	    a <a>Thing Description Directory</a> (TDD)
+	    can be used instead; in such a case the TD returned is that 
+	    of the TDD.
+	    If a small fixed number of TDs need to be returned,
+	    particularly on small devices that do not want to host the
+	    full TDD interface, a single TD can be returned, marked as 
+	    a Thing Link, which link to multiple TDs hosted  
+	    at different
+	    URLs.
```

Comment:
This gives the false impression that Thing Link and TDD are alternative ways of returning multiple TDs. But in fact, Thing Link serves the very specific purpose of allowing a single TD to **reference** one or more.